foolbox/models/base.py

```diff
@@ -64,9 +64,11 @@ def _preprocess(self, inputs: ep.TensorType) -> ep.TensorType:
         min_, max_ = self._model.bounds
         return x * (max_ - min_) + min_
 
-    @property
-    def data_format(self) -> Any:
-        return getattr(self._model, "data_format", None)
+    def __getattr__(self, item: str) -> Any:
+        if item == "data_format":
+            return getattr(self._model, "data_format")
+        else:
+            return self.__getattribute__(item)
```

Comment:
Might be better to just call `super().__getattr__` instead

Reply:
I tired this out and it didn't work:`super().__getattr__` could not be found.